<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiptManagementWidget</class>
 <widget class="QWidget" name="ReceiptManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="importTab">
      <attribute name="title">
       <string>Qu·∫£n l√Ω Phi·∫øu Nh·∫≠p</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSplitter" name="importSplitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QGroupBox" name="importListGroupBox">
          <property name="title">
           <string>Danh s√°ch Phi·∫øu Nh·∫≠p</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="createImportReceiptButton">
                <property name="text">
                 <string>‚ûï T·∫°o Phi·∫øu Nh·∫≠p M·ªõi</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="importReceiptsTable">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string>M√£ Phi·∫øu</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Ng√†y Nh·∫≠p</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Nh√† Cung C·∫•p</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T·ªïng Ti·ªÅn</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="importDetailsGroupBox">
          <property name="title">
           <string>Chi ti·∫øt Phi·∫øu Nh·∫≠p ƒë√£ ch·ªçn</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableWidget" name="importDetailsTable">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <column>
              <property name="text">
               <string>M√£ HH</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T√™n H√†ng H√≥a</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>S·ªë L∆∞·ª£ng</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>ƒê∆°n Gi√° Nh·∫≠p</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Th√†nh Ti·ªÅn</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="exportTab">
      <attribute name="title">
       <string>Qu·∫£n l√Ω Phi·∫øu Xu·∫•t</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QSplitter" name="exportSplitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QGroupBox" name="exportListGroupBox">
          <property name="title">
           <string>Danh s√°ch Phi·∫øu Xu·∫•t</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="createExportReceiptButton">
                <property name="text">
                 <string>üì§ T·∫°o Phi·∫øu Xu·∫•t M·ªõi</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="exportReceiptsTable">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string>M√£ Phi·∫øu</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Ng√†y Xu·∫•t</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T√™n Kh√°ch H√†ng</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T·ªïng Ti·ªÅn</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="exportDetailsGroupBox">
          <property name="title">
           <string>Chi ti·∫øt Phi·∫øu Xu·∫•t ƒë√£ ch·ªçn</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QTableWidget" name="exportDetailsTable">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <column>
              <property name="text">
               <string>M√£ HH</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>T√™n H√†ng H√≥a</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>S·ªë L∆∞·ª£ng</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>ƒê∆°n Gi√° B√°n</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Th√†nh Ti·ªÅn</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>